packages/airnode-validator/src/config/config.ts

```diff
@@ -153,36 +156,71 @@ export const gasPriceOracleSchema = z
 
 export const chainOptionsSchema = z
   .object({
-    fulfillmentGasLimit: z.number().int(),
+    fulfillmentGasLimit: z.number().int().optional(),
     withdrawalRemainder: amountSchema.optional(),
     gasPriceOracle: gasPriceOracleSchema,
   })
   .strict();
 
-export const ensureValidAirnodeRrp = (
-  contracts: SchemaType<typeof airnodeRrpContractSchema>,
-  chainId: string,
-  ctx: RefinementCtx
-) => {
-  if (!contracts?.AirnodeRrp) {
+export const ensureValidAirnodeRrp = (airnodeRrp: string | undefined, chainId: string, ctx: RefinementCtx) => {
```

Comment:
Won't `airnodeRrp` always be either an address string or an empty string and not undefined given how you've implemented `defaults`?

Reply:
Yes, you're right. I fixed it.